src/Grpc/test/testassets/InteropClient/InteropClient.cs

```diff
@@ -179,7 +180,19 @@ private async Task<IChannelWrapper> HttpClientCreateChannel()
             return new GrpcChannelWrapper(channel);
         }
 
-        private bool IsHttpClient() => string.Equals(options.ClientType, "httpclient", StringComparison.OrdinalIgnoreCase);
+        // TODO(JamesNK): This type can be removed in the future when Grpc.Net.Client sets VersionPolicy automatically
+        private class VersionPolicyHandler : DelegatingHandler
+        {
+            public VersionPolicyHandler(HttpMessageHandler innerHandler) : base(innerHandler)
+            {
+            }
+
+            protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+            {
+                request.VersionPolicy = HttpVersionPolicy.RequestVersionOrHigher;
```

Comment:
You plan on using Exact in gRPC, no?
```suggestion
                request.VersionPolicy = HttpVersionPolicy.RequestVersionExact;
```

Reply:
Theoretically gRPC should work with HTTP/3 so OrHigher seems like the best choice.